site/content/docs/5.3/examples/e-shop/index.html

```diff
@@ -0,0 +1,1567 @@
+        </button>
+      </li>
+      <li>
+        <button class="tag text-nowrap">
+          <svg width="1.5rem" height="1.5rem" viewBox="0 0 1000 1000" aria-hidden="true" focusable="false">
+            <use xlink:href="#new"/>
+          </svg>
+          <span class="visually-hidden">Filter by</span>New
+        </button>
+      </li>
+      <li>
+        <button class="tag text-nowrap">
+          <svg width="1.5rem" height="1.5rem" viewBox="0 0 1000 1000" aria-hidden="true" focusable="false">
+            <use xlink:href="#random"/>
+          </svg>
+          <span class="visually-hidden">Filter by</span>Refurbished
+        </button>
+      </li>
+      <li>
+        <button class="tag text-nowrap">
+          <svg width="1.5rem" height="1.5rem" viewBox="0 0 1000 1000" aria-hidden="true" focusable="false">
+            <use xlink:href="#clock"/>
+          </svg>
+          <span class="visually-hidden">Filter by</span>Soon available
+        </button>
+      </li>
+      <li>
+        <button class="tag text-nowrap">
+          <svg width="1.5rem" height="1.5rem" viewBox="0 0 1000 1000" aria-hidden="true" focusable="false">
+            <use xlink:href="#tick"/>
+          </svg>
+          <span class="visually-hidden">Filter by</span>Flash sale
+        </button>
+      </li>
+    </ul>
+    <hr class="mt-0 mt-md-3 border-1 border-light">
+    <div class="row pt-md-4 mb-3">
+
+      <div class="offcanvas-lg offcanvas-end col-lg-3 px-0 pe-lg-5 ps-lg-2" style="--bs-offcanvas-width: 280px;" tabindex="-1" id="filterOffcanvas" aria-labelledby="filterOffcanvasLabel">
+        <div class="offcanvas-header border-bottom border-1 border-light">
+          <h5 class="offcanvas-title fs-5" id="filterOffcanvasLabel">Filter 'Smartphones'</h5>
+          <button type="button" class="btn-close" data-bs-dismiss="offcanvas" data-bs-target="#filterOffcanvas"><span class="visually-hidden">Close</span></button>
+        </div>
+        <div class="offcanvas-body">
+
+          <form class="d-flex flex-column">
+            <div class="mb-2 pt-lg-4">
+              <h3 class="h4 mb-0">Purchase type</h3>
+              <hr class="border-1 border-light my-2">
+              <div class="d-flex flex-column">
+                <div class="form-check mb-2">
+                  <input class="form-check-input" type="checkbox" value="" id="offerTypeFilter">
+                  <label class="form-check-label" for="offerTypeFilter">
+                    With offer (56)
+                  </label>
+                </div>
+                <div class="form-check mt-1">
+                  <input class="form-check-input" type="checkbox" value="" id="mobileTypeFilter" checked>
+                  <label class="form-check-label" for="mobileTypeFilter">
+                    Mobile only (42)
+                  </label>
+                </div>
+              </div>
+            </div>
+
+            <div class="mb-2 pt-4">
+              <h3 class="h4 mb-0">Brands</h3>
```

Comment:
Categories (brands, colors,..) should be grouped by a fieldset for a11y purposes:

```
<fieldset>
  <legend>Brands</legend>
  [brands ...]
</fieldset>
<fieldset>
  <legend>Colors</legend>
  [colors ...]
</fieldset>
```

Or at least:

```
<div role="group" aria-labelledby="brands">
  <p id="brands">Brands</p>
  [brands ...]
</div>
<div role="group" aria-labelledby="colors">
  <p id="colors">Colors</p>
  [colors ...]
</div>
```